src/Facebook/InstantArticles/Elements/Element.php

```diff
@@ -86,4 +88,30 @@ protected function emptyElement($document)
         $fragment->appendChild($document->createTextNode(''));
         return $fragment;
     }
+
+    /**
+     * Method that returns the status of ignoring empty for this Element.
```

Comment:
Please comment what this actually do: "Suppress the empty tag warning on validation".